src/main/java/com/hubspot/jinjava/util/ObjectTruthValue.java

```diff
@@ -39,7 +39,19 @@ public static boolean evaluate(Object object) {
     }
 
     if (object instanceof Number) {
```

Comment:
There are more [subclasses of `Number`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Number.html). While you don't have to handle them all, perhaps a sane default would be to use `Number.floatValue()`.